client/src/layers/Kanban/index.tsx

```diff
@@ -5,41 +5,52 @@ import KanbanModal from '@/components/KanbanModal';
 import { StoryType, StatusType } from '@/types/story';
 import { useStoryState } from '@/lib/hooks/useContextHooks';
 
-export interface KanbanDefaultType {
-  handleDragStart(e: React.DragEvent<HTMLElement>, order: number, category: StatusType): void;
-  handleDragEnter(e: React.DragEvent<HTMLElement>, order: number, category: StatusType): void;
-}
-
 const Kanban = () => {
   const draggingRef = useRef<number | null>(0);
   const dragOverRef = useRef<number | null>(0);
-  const cateogryRef = useRef<StatusType>('TODO');
+  const draggingCategory = useRef<StatusType>('TODO');
+  const dragOverCateogry = useRef<StatusType>('TODO');
   const storyList: StoryType[] = useStoryState();
 
   const todoList = storyList
     .filter((item) => item.status === 'TODO')
-    .sort((a, b) => a.order - b.order);
+    .sort((a, b) => Number(a.order) - Number(b.order));
   const onGoingList = storyList
     .filter((item) => item.status === 'IN_PROGRESS')
-    .sort((a, b) => a.order - b.order);
+    .sort((a, b) => Number(a.order) - Number(b.order));
   const finishList = storyList
     .filter((item) => item.status === 'DONE')
-    .sort((a, b) => a.order - b.order);
+    .sort((a, b) => Number(a.order) - Number(b.order));
 
   return (
     <KanbanModal>
       <Styled.Container>
         <Styled.Title>프로젝트 칸반보드</Styled.Title>
         <Styled.ColumnContainer>
           <KanbanColumn
-            draggingRef={draggingRef}
-            dragOverRef={dragOverRef}
-            categoryRef={cateogryRef}
             category={'TODO'}
             storyList={todoList}
+            draggingRef={draggingRef}
+            dragOverRef={dragOverRef}
+            draggingCategory={draggingCategory}
+            dragOverCategory={dragOverCateogry}
+          />
+          <KanbanColumn
+            category={'IN_PROGRESS'}
+            storyList={onGoingList}
+            draggingRef={draggingRef}
+            dragOverRef={dragOverRef}
+            draggingCategory={draggingCategory}
+            dragOverCategory={dragOverCateogry}
+          />
+          <KanbanColumn
+            category={'DONE'}
+            storyList={finishList}
+            draggingRef={draggingRef}
+            dragOverRef={dragOverRef}
+            draggingCategory={draggingCategory}
+            dragOverCategory={dragOverCateogry}
```

Comment:
dragging- 가 계속 반복되는데 이렇게 하면 어떨까요? 너무 억지인가요??

```suggestion
          [
            { category: 'TODO', storyList: todoList },
            { category: 'IN_PROGRESS', storyList: onGoingList },
            { category: 'DONE', storyList: finishList },
          ].map(({ category, storyList }) => (
            <KanbanColumn
              category={category}
              storyList={storyList}
              draggingRef={draggingRef}
              dragOverRef={dragOverRef}
              draggingCategory={draggingCategory}
              dragOverCategory={dragOverCateogry}
            />
          ));
```